CMakeLists.txt

```diff
@@ -334,6 +334,7 @@ if(SLANG_ENABLE_SLANG_GLSLANG)
         MODULE
         USE_FEWER_WARNINGS
         LINK_WITH_PRIVATE glslang SPIRV SPIRV-Tools-opt
+        INCLUDE_DIRECTORIES_PUBLIC include
```

Comment:
Since this is only used in slang-glslang source (and not its header) this can be INCLUDE_DIRECTORIES_PRIVATE

Reply:
act